.travis.yml

```diff
@@ -106,10 +140,60 @@ script:
         hasIosFile=$(npm run danger -- run --dangerfile ./dangerfile-ios.js)
         echo "The value of hasIosFile is ${hasIosFile}"
         if [[ "$hasIosFile" =~ "hasIosFile" ]]; then
-          xcodebuild -project ios/sdk/WeexSDK.xcodeproj test -scheme WeexSDKTests CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination "platform=iOS Simulator,name=iPhone 6"
+          # build WeexSDK and run WeexSDKTests
+          xcodebuild -project ios/sdk/WeexSDK.xcodeproj test -scheme WeexSDKTests CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination "platform=iOS Simulator,name=iPhone 6" || exit 1
+          # build WeexDemo and run WeexDemo test
+          cd weex-playground/ios && mkdir tmp && mv * tmp;cd tmp
+          xcodebuild -workspace WeexDemo.xcworkspace test -scheme WeexDemo CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination "platform=iOS Simulator,name=iPhone 6" || exit 1
         fi
         ;;
     esac
+    
+    if [[ ("${STATIC_CODE_ANALYSIS}" = "true") && ("${CHECK_C}" = "true") ]]; then
+      hasCFile=$(npm run danger -- run --dangerfile ./dangerfile-static-check.js)
+      echo "The value of hasCFile is ${hasCFile}"
+      if [[ "$hasCFile" =~ "hasCFile" ]]; then
+        echo "hasCFile"
+        cd ios/sdk && xcodebuild | xcpretty -r json-compilation-database -o compile_commands.json
+        oclint-json-compilation-database oclint_args -- \
```

Comment:
You need to output the result within danger, `stdout` is not useless in current situation.

Reply:
do you mean `echo "hasCFile"`, in this case, it just means the ci have executed this step.

oclint output is too long,I think we shouldn't output oclint log

Reply:
But if you didn't out the result of the oc_lint through `danger`, there is no way for contributors to know what is wrong. Generally no one will read the raw log of Travis if build is success.

Reply:
yes, generally no one will read the raw log of Travis if build is success, but this also means that oclint only output warning message.

but the oclint logs is over 1000 lines ,if we put them in danger, is too long !

maybe we can update the input file of oclint from the whole `ios-sdk` project to the file that changed of pr.or we can change the oclint rules to 4 rules may lead to bug or crash.in this case, the log will be short enougn to output by danger.